Program.cs

```diff
@@ -6,52 +6,62 @@
 using System.Threading.Tasks;
 using System;
 using System.Net.Http.Headers;
+using System.Runtime.CompilerServices;
 
 namespace Lesson
 {
     class Program
     {
         static void Main(string[] args)
         {
-            double a, b;
-            string operation;
-
+            double a = 0.0, b = 0.0;
```

Comment:
Zmien nazwy zmiennych.
Nie musisz inicjalizowac zmiennych, czyli dawac im poczatkowych wartosci. Defaultowo one posiadaja wartosc 0.0.

Reply:
1)            double a, b;
            Console.WriteLine("kalkulator Chopie");

            Console.WriteLine("Type figure a");
            a = IsNumber(a);

            Console.WriteLine("Type figure b");
            b = IsNumber(b);
            
Error	CS0165	Use of unassigned local variable 'a'
Error	CS0165	Use of unassigned local variable 'b'

2)          Console.WriteLine("kalkulator Chopie");

            Console.WriteLine("Type figure a");
            var a = IsNumber(a);

            Console.WriteLine("Type figure b");
            var b = IsNumber(b);

Error	CS0841	Cannot use local variable 'a' before it is declared
Error	CS0841	Cannot use local variable 'b' before it is declared

That's why.

Reply:
pewnie w tej funkcji coś nie teges, jeśli przy tym można użyć var: var operation = Console.ReadLine();